diffpy/pdfmorph/pdfplot.py

```diff
@@ -217,16 +222,17 @@ def truncatePDFs(r, gr, rmin = None, rmax = None):
 def _configure():
     """Configure look and feel."""
     import matplotlib.pyplot as plt
-    plt.rc("font", size = 40)
-    plt.rc("axes", linewidth = 2, labelsize = 30)
-    plt.rc("xtick", labelsize = 25)
-    plt.rc("xtick.major", size = 7)
-    plt.rc("xtick.minor", size = 3)
-    plt.rc("ytick", labelsize = 25)
-    plt.rc("ytick.major", size = 7)
-    plt.rc("ytick.minor", size = 3)
-    plt.rc("legend", fontsize = 18)
-    plt.rc("lines", markeredgewidth = 2) # thicker axes and symbols
+    plt.style.use(bg_mpl_style)
+    plt.rc("font") #, size = 40)
```

Comment:
I think you can delete all these as they should be being handled in the style-sheet.  If there are any things we want to override (such as the legend fontsize) we can leave that in, but keep it to a minimum.  In fact, as you ahve done, switch them all off (but do it by deleting them) then only add them back in as needed on esthetic grounds.

Reply:
removed, legend was handled separately already and I reintroduced it, it was commented out previously